packages/core/src/components/TextArea/index.tsx

```diff
@@ -58,7 +62,14 @@ export default class TextArea extends React.Component<Props, State> {
     return (
       <FormField {...fieldProps} id={id} labelDescription={description}>
         {proofread ? (
-          <Proofreader {...(textareaProps as any)} id={id} locale={locale} />
+          <Proofreader
+            {...(textareaProps as any)}
+            {...proofreadProps}
+            id={id}
+            locale={locale}
+            isRuleHighlighted={proofreadProps && proofreadProps.isRuleHighlighted}
```

Comment:
You don't need these 2 lines if you're spreading `proofreadProps` above.

Reply:
Oh that's right, thanks!